io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,183 @@
+          "sha256": "85cf747b5923f18a8d0f37d0cef647636d70ddfeda5de49ea019e7e440f9e4e0"
+        },
+        {
+          "type": "file",
+          "dest-filename": "lpub3dldrawunf.zip",
+          "url": "https://github.com/trevorsandy/lpub3d_libs/releases/download/v1.0.1/lpub3dldrawunf-01-04-21.zip",
+          "sha256": "f0266006c4b7114e4f4719801696fbe8780cf9d767e948fb6478c752989cbee8"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/trevorsandy/lpub3d_libs/releases/download/v1.0.1/tenteparts.zip",
+          "sha256": "7e1a68490b1be11bddcfb3180151b42ffbb85ce3aa8ec3f7431bee5c04cc1ac5"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/trevorsandy/lpub3d_libs/releases/download/v1.0.1/vexiqparts.zip",
+          "sha256": "e2f4114c8ff6bfe7e5a43b6c88fb47f32328ddb147578456ec0825aa76803b20"
+        }
+      ],
+      "post-install": [
+        "mv -f complete.zip ${LP3D_3RD_PARTY_DIR}/complete.zip",      
+        "mv -f lpub3dldrawunf.zip ${LP3D_3RD_PARTY_DIR}/lpub3dldrawunf.zip",
+        "mv -f tenteparts.zip ${LP3D_3RD_PARTY_DIR}/",
+        "mv -f vexiqparts.zip ${LP3D_3RD_PARTY_DIR}/"
+      ]
+    },
+    {
+      "name": "ldglite",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "CONFIG+=3RD_PARTY_INSTALL=${LP3D_3RD_PARTY_DIR}/ldglite_renderer",
+        "CONFIG+=release",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldglite.git",
+          "tag": "v1.3.6",
+          "commit": "140865cb85b17eac1abe613066c9364db1e57650"
+        }
+      ],
+      "post-install": [
+        "mv -f ${LP3D_3RD_PARTY_DIR}/ldglite_renderer/* ${LP3D_3RD_PARTY_DIR}/"
+      ]
+    },
+    {
+      "name": "ldview",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "CONFIG+=3RD_PARTY_INSTALL=${LP3D_3RD_PARTY_DIR}/ldview_renderer",
+        "CONFIG+=release",
+        "CONFIG+=BUILD_CUI_ONLY",
+        "CONFIG+=USE_SYSTEM_LIBS",
+        "CONFIG+=BUILD_TINYXML",
+        "CONFIG+=BUILD_GL2PS",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldview.git",
+          "branch": "qmake-build",
+          "commit": "16b44f3658690294b1faf13778e3654d443e3751"
+        }
+      ],
+      "post-install": [
+        "mv -f ${LP3D_3RD_PARTY_DIR}/ldview_renderer/* ${LP3D_3RD_PARTY_DIR}/"
+      ]
+    },
+    {
+      "name": "povray",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd unix && ./prebuild3rdparty.sh",
+        "./configure COMPILED_BY=\"Trevor SANDY <trevor.sandy@gmail.com> using FlatHub for LPub3D.\" --prefix=${LP3D_3RD_PARTY_DIR}/povray_renderer LPUB3D_3RD_PARTY=yes --with-libsdl2 --with-boost-libdir=${FLATPAK_DEST}/lib --enable-watch-cursor",
```

Comment:
doesn't this configure line indicate you need SDL2? If yes then it'll need to be built as well.

Reply:
Dependencies. Yep. This was the area I wanted to get to so I could determine the next step. 

There was no intuitive way to determine what libraries were available (or unavailable) in the runtime and; moreover, no intuitive way to simply list the off-the-shelf dependencies. If, i well understand, I’ll have to go find each needed dependency download, find or generate its sha256 and manually add it to the build ?

Cheers,

Reply:
The entirety of my [AppImage yml](https://github.com/trevorsandy/lpub3d-ci/blob/master/builds/linux/obs/alldeps/appimage.yml) is 60 lines - listing the required dependencies is very intuitive. 

Does flatpack have anything similar?

Cheers,

Reply:
no.